packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -1049,6 +1049,8 @@ export class PostgresEngineVersion {
 
   /** Version "15" (only a major version, without a specific minor version). */
   public static readonly VER_15 = PostgresEngineVersion.of('15', '15', { s3Import: true, s3Export: true });
+  /** Version "15.2". */
+  public static readonly VER_15_2 = PostgresEngineVersion.of('15.2', '15', { s3Import: true, s3Export: true });
```

Comment:
if there's a 15.2, there must be a 15.1. Could we add that one as well?

Reply:
15.2 is the first minor version of PostgreSQL 15  supported by RDS
https://docs.aws.amazon.com/AmazonRDS/latest/PostgreSQLReleaseNotes/postgresql-versions.html#postgresql-versions-version15